code/game/machinery/machinery.dm

```diff
@@ -325,6 +325,15 @@ Class Procs:
 /obj/machinery/CouldNotUseTopic(var/mob/user)
 	usr.unset_machine()
 
+
+/obj/machinery/proc/dropContents()//putting for swarmers, occupent code commented out, someone can use later.
+	var/turf/T = get_turf(src)
+	T.contents += contents

```

Comment:
i tried to call it two diffrent ways for force move and got an undefined proc error.